File: Server/ProtocolCodes.py

class RequestType(object):
    download = 1
    exit = 0
    upload = 2

class ErrorCode(object):
    fail = 1
    fileexists = 4
    ok = 0
    wrongtype = 2
    invalidpath = 3

class FileType(object):
    directory = 1
    file = 0

File: Client/Android/app/src/main/java/ozelentok/zbackup/ProtocolCodes.java
package ozelentok.zbackup;

public class ProtocolCodes {
	public enum RequestType {
		exit,
		download,
		upload
	}
	public enum ErrorCode {
		ok,
		fail,
		wrongtype,
		invalidpath,
		fileexists,
	}
	public enum FileType {
		file,
		directory
	}

}
